spec/requests/resources_spec.rb

```diff
@@ -32,4 +32,14 @@
       end
     end
   end
+  context 'show' do
+    let!(:resources) { create_list :resource, 4 }
+    let!(:resource_a) { create :resource, name: 'a' }
+
+    it 'returns specific resource' do
+      get "/resources/#{resource_a.id}"
+      returned_ids = response_json['id']
+      expect(returned_ids).to match(resource_a.id)
```

Comment:
is that syntax correct? not familiar with rspec matchers but I can't find out how that would match against the reference just containing an Array of anything